src/ui/public/courier/search_source/__tests__/search_source.js

```diff
@@ -49,16 +49,16 @@ describe('SearchSource', function () {
     indexPattern2 = new IndexPattern('test2-*', null, []);
     expect(indexPattern).to.not.be(indexPattern2);
   }));
-  beforeEach(requestQueue.clear);
-  after(requestQueue.clear);
+  beforeEach(searchRequestQueue.removeAll);
```

Comment:
nit: If you did here what you did in https://github.com/elastic/kibana/pull/20332/files#diff-34a3c2fb9db6923637e99898043c210aR28 then we wouldn't need to make all the methods on the `SearchRequestQueue` class bound arrow functions anymore.